dotnet/samples/04.e.twentyQuestions/README.md

```diff
@@ -0,0 +1,113 @@
+# AI in Microsoft Teams: Twenty Qestions
+
+Welcome to the 20 Questions Bot: The Ultimate Guessing Game! This developer sample application showcases the incredible capabilities of language models and the concept of user intent. Challenge your skills as the human player and try to guess a secret within 20 questions, while the AI-powered bot answers your queries about the secret. Experience firsthand how language models interpret user input and provide informative responses, creating an engaging and interactive gaming experience. Get ready to dive into the world of language models and explore the fascinating realm of user interaction and intent.
+It shows following SDK capabilities:
+
+<details open>
+    <summary><h3>Bot scaffolding</h3></summary>
+    Throughout the 'Program.cs' and 'GameBot.cs' files you'll see the scaffolding created to run a bot with AI features.
+</details>
+<details open>
+    <summary><h3>Prompt engineering</h3></summary>
+The 'Prompts/Hint/skprompt.txt' file has descriptive prompt engineering that, in plain language, instructs GPT how the bot should conduct itself at submit time. For example, in 'skprompt.txt':
+
+#### skprompt.txt
+
+```text
+You are the AI in a game of 20 questions. 
+The goal of the game is for the Human to guess a secret within 20 questions. 
+The AI should answer questions about the secret.
+The AI should assume that every message from the Human is a question about the secret.
+
+GuessCount: {{$guessCount}}
+RemainingGuesses: {{$remainingGuesses}}
+Secret: {{$secretWord}}
+
+Answer the humans question but do not mention the secret word.
+
+Human: {{$input}}
+AI: 
+```
+
```

Comment:
General comment for all samples: If we are adding the prompt text to the README, we should do a better job of explaining what each section does (directions, entities, history)

Reply:
Added explanation.
Also, I think we can add link here after the .NET doc is ported, e.g., prompt templating with .NET SDK.